src/factories/interfaces/IPIM_WorkflowFactory_v1.sol

```diff
@@ -23,6 +23,7 @@ import {IERC20} from "@oz/token/ERC20/IERC20.sol";
 interface IPIM_WorkflowFactory_v1 {
     //--------------------------------------------------------------------------
     // Errors
+    /// @notice Error thrown when an unpermissioned address tries to claim fees to to transfer role.
```

Comment:
`tries to claim fees to to transfer role` seems like it contains a `to` too much? :D